meson.build

```diff
@@ -529,10 +529,12 @@ endif
 
 root_incdir = include_directories('.')
 
-if get_option('gtkdoc')
-  gtkdocscan = find_program('gtkdoc-scan', required : true)
-  subdir('docs')
-endif
+gidocgen_dep = dependency(
+  'gi-docgen', version: '>= 2021.1',
+  fallback: ['gi-docgen', 'dummy_dep'],
+  required: get_option('gtkdoc'),
+)
+
```

Comment:
shouldn't this be only if gtkdoc is set to true? Otherwise it will cause it to subproject on all builds